README.md

```diff
@@ -103,7 +103,7 @@ looks like in practice.
 
 Why is this library called `html5gum`?
 
-* G.U.M: **G**iant **U**nreadable **M**atch-statement
+* G.U.M: [**G**iant **U**nreadable **M**achine][src/machine.rs]
```

Comment:
can we show both (`giant unreadable machine/giant unreadable match-statement`), I guess I am trying to preserve the historical context here